### PR TITLE
Hero - Donations Page

### PR DESCRIPTION
**Pull request by radhikap0107**
Initial changes to Hero Section for donations page

### CODE REVIEW

File: donate.html [hunk 1]
```diff
@@ -123,9 +129,22 @@
...
 
+  <div class="hero">
+    <div class="container">
+      <div class="hero-information">
+        <div class="hero-logo">
+          <img id="technica-logo" src="img/technica-logo.png" alt="Technica logo">
+          </div>
+      </div>
+      <div class="hero-action text-center">
+        <button class="donate-btn btn" onclick="location.href='https://register.gotechnica.org'">Donate</button>
```
Comment (julesparra) [inline donate.html:140]: would make donate-btn bigger in ur scss to pop a bit more

File: scss/donate.scss [hunk 2]
```diff
@@ -1,5 +1,166 @@
 /*@use 'vars';*/
 //temporary until resolve _vars.scss issue
+
+// HERO SECTION BEGINS HERE
+// .hero-clouds {
+//   position: absolute;
+//   width: 100%;
+//   height: 100vh;
+//   z-index: 1;
+//   @media only screen and (max-width: 767px) {
```
Comment (julesparra) [inline scss/donate.scss:10]: be sure to test mobile responsiveness, looks like the hero covers more of the sections underneath when I make the screen smaller

**Review by julesparra — CHANGES_REQUESTED** [review]
good start! left some comments on things to think through a bit more

File: donate.html [hunk 3]
```diff
@@ -125,24 +131,18 @@
...
-          <span style="position: relative">
-          Donating to
-          <div class="d-none d-md-block">
-            <div class="large-spark pink-spark" style="top: -30px; left: -70px;"></div>
-            <div class="med-spark purp-spark" style="top: 50px; left: -120px;"></div>
-            <div class="small-spark blue-spark" style="top: 190px; left: -50px;"></div>
-            <div class="large-spark purp-spark" style="top: -35px; right: -40px;"></div>
-            <div class="small-spark pink-spark" style="top: 210px; right: -20px;"></div>
-            <div class="med-spark blue-spark" style="top: 90px; right: -110px;"></div>
-          </div>
```
Comment (julesparra) [inline donate.html:140]: i suppose this was copied over from the old version, but the new design doesn't have any sparkles so can delete

File: donate.html [hunk 4]
```diff
@@ -125,24 +131,18 @@
...
-        <img src="img/technica-logo.png" alt="Technica orb logo" class="top-logo" width="400" height="90">
-        <div class="b1"> <a target="_blank" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url" class="donate-button">Donate</a> </div>
+      <div class="hero-information">
+        <h1> Donating to </h1> -->
+        <div class="hero-logo">
+          <img id="technica-logo" src="img/technica-logo.png" alt="Technica logo" ALIGN = "top"> 
+        </div>
       </div>
+      <div class="hero-action text-center">
+        <button class="register-btn btn" onclick="location.href='https://register.gotechnica.org'">Donate</button>
```
Comment (julesparra) [inline donate.html:143]: let's change the register-btn class to donate-btn and direct css to that class

File: donate.html [hunk 5]
```diff
@@ -125,24 +131,18 @@
...
-            <div class="large-spark pink-spark" style="top: -30px; left: -70px;"></div>
-            <div class="med-spark purp-spark" style="top: 50px; left: -120px;"></div>
-            <div class="small-spark blue-spark" style="top: 190px; left: -50px;"></div>
-            <div class="large-spark purp-spark" style="top: -35px; right: -40px;"></div>
-            <div class="small-spark pink-spark" style="top: 210px; right: -20px;"></div>
-            <div class="med-spark blue-spark" style="top: 90px; right: -110px;"></div>
-          </div>
-        </span>
-      </h2>
-        <img src="img/technica-logo.png" alt="Technica orb logo" class="top-logo" width="400" height="90">
```
Comment (julesparra) [inline donate.html:143]: i see you added a Donations_technica_logo png to the img folder - don't need that, doesn't look like you're using it anyway

File: donate.html (hunk 4 above)
Comment (julesparra) [reply donate.html:143]: also looks like this is linking to the registration portal, replace with paypal link: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url

File: scss/donate.scss [hunk 6]
```diff
@@ -1,36 +1,173 @@
...
-  text-align: center;
-  color: #fff;
-  border: none;
-  background-color: #ff5789;
-  align-items: center;
-  padding: 7px 35px;
-  font-size: 35px;
-  font-weight: bold;
-  border-radius: 6px;
+HERO SECTION BEGINS HERE
```
Comment (julesparra) [inline scss/donate.scss:4]: comment this out

File: scss/donate.scss [hunk 7]
```diff
@@ -1,36 +1,173 @@
...
+
+  @media only screen and (min-width: 996px) {
+    padding-top: 115px;
+  }
+
+  @media only screen and (min-width: 1400px) {
+    padding-top: 350px;
+  }
+
+  .register-btn {
```
Comment (julesparra) [inline scss/donate.scss:103]: (also rename) donate button big too big & text doesn't scale with size of button on mobile
Screen Shot 2021-06-24 at 12 13 54 AM

File: donate.html [hunk 8]
```diff
@@ -125,24 +131,18 @@
...
-        <div class="b1"> <a target="_blank" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url" class="donate-button">Donate</a> </div>
+      <div class="hero-information">
+        <h1> Donating to </h1> -->
```
Comment (julesparra) [inline donate.html:137]: "-->" needs to be deleted

File: scss/donate.scss [hunk 9]
```diff
@@ -1,36 +1,173 @@
...
 }
 
-.donate-button:hover {
-  opacity: 75%;
-  text-decoration: none;
-  color: #fff;
-  transition: 0.3s ease;
+.hero {
+  position: relative;
+  overflow: hidden;
```
Comment (julesparra) [inline scss/donate.scss:27]: "donating to" above the technica logo disappears on mobile, `overflow: hidden` is likely the culprit, delete
Screen Shot 2021-06-24 at 12 13 54 AM

**Review by julesparra — CHANGES_REQUESTED** [review]
getting there🌸 lmk if u need any help testing responsiveness:)

File: donate.html (hunk 4 above)
Comment (radhikap0107) [reply donate.html:143]: Oops yes I was gonna ask you about the actual link but totally forget

**Pull request closed without merging**